AcademicReward/Database/IDatabase.cs

```diff
@@ -11,6 +13,7 @@ public interface IDatabase {
         public DatabaseErrorType UpdateItem(object obj);
         public DatabaseErrorType DeleteItem(object obj);
         public DatabaseErrorType LookupItem(object obj);
-        public DatabaseErrorType LookupFullItem(object obj); 
+        public DatabaseErrorType LookupFullItem(object obj);
+        public DatabaseErrorType LoadItems(ObservableCollection<object> obj, string[] args);
```

Comment:
Populates an observable collection with database data, good if you don't need a logic layer (like with History in my case)